### PR TITLE
publish to docker

### PR DESCRIPTION
**Pull request by beemarie**

### CODE REVIEW

**Comment by beemarie** [comment]
Closes #597

File: .travis.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ after_success:
 - DEPLOY_BUILD_READY=true
 after_script:
 - make clean
+- ./tools/travis/docker/publish.sh apache wskdeploy ${TRAVIS_TAG}
```
Comment (houshengbo) [inline .travis.yml]: We need to move this line into a new section of provider.

deploy:
-provider: releases
 ...
 ...
- provider: script
  script: "./tools/travis/docker/publish.sh apache wskdeploy ${TRAVIS_TAG}"
  on:
    tags: true
    repo: apache/incubator-openwhisk-wskdeploy
    condition: "$DEPLOY_BUILD_READY = true"

File: tools/travis/docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FROM openwhisk/nodejs6action
```
Comment (houshengbo) [inline tools/travis/docker/Dockerfile]: Do we only expect nodejs?

Comment (beemarie) [reply tools/travis/docker/Dockerfile]: yes, we're building on top of this dockerfile to run the package action.

File: .travis.yml (hunk 1 above)
Comment (beemarie) [reply .travis.yml]: thanks -- this wasn't originally working, until I realized i needed skip_cleanup = true.  Should be good to go now.

**Pull request merged**